src/ontology/pato-edit.obo

```diff
@@ -21847,6 +21847,40 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
 is_a: PATO:0002248 ! pigmented
 created_by: https://orcid.org/0000-0002-3528-5267
 
+[Term]
+id: PATO:0070002
+name: basket cell morphology
+def: "A cell morphology that inheres in neurons that have sparsely spinous dendrites, which spread radially and tangentially, with axons on the pial side of the soma ascending or descending to give off horizontal collaterals, which in turn sprout small branches ending in dense tufts of terminals." [doi:10.1016/b978-0-12-369497-3.10004-4]
+xref: ILX:0101123
+xref: ilxtr:BasketPhenotype
+xref: nifext:3
+is_a: PATO:0010006 ! cell morphology
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070003
+name: small basket cell morphology
+def: "A basket cell morphology that inheres in neurons that are aspiny and multipolar, with axonal arbors, composed of frequent short, curvy axonal branches, that tend to be near their somata and within the same layer." [PMID:11884355]
+xref: ilxtr:SmallBasketPhenotype
```

Comment:
'aspiny' seems to contradict assertion on parent class 'sparsely spinous'.  Maybe 'basket cell' should be defined as aspiny of sparsely spiny.

Punctuation needs fixing:

> with axonal arbors, composed of frequent short, curvy axonal branches, that tend to be near their somata and within the same layer.

-> 
with axonal arbors composed of frequent, short, curvy axonal branches that tend to be near their somata and within the same layer." [PMID:11884355]

Reply:
Fixed